src/Kernel/client/kernel.ts

```diff
@@ -223,13 +223,15 @@ class Kernel {
             return;
         }
 
-        var isLocalEnvironment =
+        const isLocalEnvironment =
             location.hostname == "localhost"
             || location.hostname == "127.0.0.1"
             || this.hostingEnvironment == null
             || this.hostingEnvironment == "";
+        const enableClientTelemetry =
```

Comment:
[nit] variable naming - I'd suggest something like `forceEnableClientTelemetry`, since this is not the only way for client telemetry to be enabled (i.e., a value of `false` does not necessarily mean that client telemetry is disabled).

Reply:
Makes sense. I renamed it.